test/AccountManager.test.ts

```diff
@@ -98,6 +98,42 @@ describe('AccountManager', function () {
       });
     });
 
+    describe('removeAccount()', function () {
+      let fakeAccounts: Array<Wallet>;
```

Comment:
Actually they're real accounts
```suggestion
      let randomAccounts: Array<Wallet>;
```

Reply:
Done.